tests/job_enable_common.erl

```diff
@@ -0,0 +1,572 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(job_enable_common).
+
+% Public API for use by other tests
+-export([
+    bin_bucket/1,
+    bin_key/1,
+    bin_val/1,
+    close_client/1,
+    enabled_string/1,
+    get_enabled/2,
+    index_2i/0,
+    index_name/1,
+    index_yz/0,
+    load_data/1,
+    num_buckets/0, num_buckets/1,
+    num_keys/0, num_keys/1,
+    open_client/2,
+    populated_bucket/0,
+    set_enabled/3,
+    setup_cluster/1,
+    setup_yokozuna/1,
+    test_buckets/0,
+    test_keys/0,
+    test_nums/0,
+    test_operation/4,
+    test_vals/0,
+    undefined_bucket/0
+]).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("riakc/include/riakc.hrl").
+-include_lib("riakhttpc/include/rhc.hrl").
+-include("job_enable_common.hrl").
+
+-define(DEFAULT_NUM_BUCKETS,    7).
+-define(DEFAULT_NUM_KEYS,       9).
+
+%% ===================================================================
+%% Test API
+%% ===================================================================
+
+enabled_string(true) ->
+    "enabled";
+enabled_string(false) ->
+    "disabled".
+
+bin_bucket(Num) ->
+    erlang:list_to_binary(["Bucket_", erlang:integer_to_list(Num)]).
```

Comment:
Alright well...since we have zero established coding standards I suppose we'll just have to agree to disagree ;). If this were code in an existing module that used the unqualified style, I'd say we shouldn't introduce style inconsistencies, but since it's a new module I guess at least we're being consistent.